templates/daemon.systemd.erb

```diff
@@ -2,7 +2,7 @@
 [Unit]
 Description=Prometheus <%= @name %>
 Wants=basic.target
-After=basic.target network.target
+After=basic.target network.target<% @unit_after.each do |after_item| %> <%= after_item %><% end %>
```

Comment:
are you by any chance interested to migrate this template from erb to epp?

Reply:
I have made this suggested change. This is my first time working with epp, so I'm not sure if there are better approaches than what I have here.